actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -254,6 +254,21 @@ private void createUser(Request actorMessage) {
         return;
       }
     }
+    if (userMap.containsKey(JsonKey.ORG_EXTERNAL_ID)) {
```

Comment:
Check and throw error if org_external_id and organisationid both are submitted

Reply:
You can use parameterMismatch error code.